app/models/fika.rb

```diff
@@ -0,0 +1,5 @@
+class Fika < ApplicationRecord
+  validates_presence_of :start_date, :end_date
+
+  has_and_belongs_to_many :fika_goers, join_table: 'fikas_fika_goers'
```

Comment:
Everything looks great except this hahaha

Reply:
but I dont have a better solution....

Reply:
atm

Reply:
Hopefully solved by changing `fika_goer` to `participant`